BioCro/DARPA/biocro_regional_darpa.md

```diff
@@ -0,0 +1,132 @@
+How to Do Regional Runs on DARPA Data Using BioCro Model
+================
+Kristina Riemer, University of Arizona
+
+## Read in libraries
+
+``` r
+library(ncdf4)
+library(PEcAn.all)
+library(PEcAn.BIOCRO)
+library(dplyr)
+library(ncdf.tools)
+library(ggplot2)
+library(maps)
+library(gganimate)
+```
+
+## Read in input files
+
+Past run config
+file:
+
+``` r
+config <- read.biocro.config("biocro_darpa_results/run/SA-median/config.xml")
+```
+
+Data files:
+
+``` r
+metfile <- "biocro_regional_files/champaign.nc"
+met_champaign <- nc_open(metfile)
+soil_nc <- nc_open("biocro_regional_files/hwsd.nc")
+```
+
+## Convert weather data
+
+The weather data is a subset of NARR around Champaign, Illinois. It’s
+currently in the Pecan standard format, which is a netCDF file. The
+dimensions are 3 latitude, 6 longitude, and 93,504 time steps, which are
+hourly from 1979 through 2010. There are various weather variables,
+including temperature, precipitation flux, and solar radiation.
+
+This is converted to the met format for BioCro. Data for each of the
```

Comment:
Is `met` an acronym? Is it capitalized in written text? (No need to answer, just posing the question)

Perhaps: `This is converted ...` -> `The netCDF file format needs to be converted ...`

Reply:
Met is short for meteorology.